### ISSUE TITLE
Failing test MessageFactoryTest::testCreationOfHtmlMessageWithAutomaticPlainText

### ISSUE THREAD

**Issue by ohvitorino** [sumocoders/Framework#148]
The test `testCreationOfHtmlMessageWithAutomaticPlainText` is failing with the following error:

```
--

There was 1 failure:

1) SumoCoders\FrameworkCoreBundle\Tests\Mail\MessageFactoryTest::testCreationOfHtmlMessageWithAutomaticPlainText
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
 '<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
 <html>
-  <head></head>
-  <body>
-    <p>And I, le content</p>
-  </body>
-</html>
-'
+<head></head>
+<body><p>And I, le content</p></body>
+</html>'

/var/www/html/src/SumoCoders/FrameworkCoreBundle/Tests/Mail/MessageFactoryTest.php:109
```

The expected value doesn't match the value being mocked in

```
    // SumoCoders/FrameworkCoreBundle/Tests/Mail/MessageFactoryTest.php
    /**
     * @return \PHPUnit_Framework_MockObject_MockObject
     */
    protected function getTemplatingMock()
    {
        $templating = $this->getMock('\Symfony\Bundle\FrameworkBundle\Templating\EngineInterface');
        $templating
            ->method('render')
            ->willReturn('<html><head></head><body><p>And I, le content</p></body></html>');

        return $templating;
    }
```

**Comment by StijnVrolijk** [comment]
Looks like @dmoerman isn't the only one with this issue :p

**Comment by jonasdekeukelaere** [comment]
Fixed in #162 - https://github.com/sumocoders/Framework/commit/d4ef5aead31760c664987de48f2b27194bfbbb9c
